src/commonmark/en/content/implementation/support.md

```diff
@@ -7,18 +7,17 @@ This chapter will describe these in more detail.
 
 ## Home page: dhis2.org
```

Comment:
Maybe we should re-write this whole section in a way that is easier to maintain. Right now, describing what is on each page is a bit of a maintenance nightmare.